pootle/templates/browser/_actions.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <ul>
+  {% if not can_suggest_only %}
```

Comment:
@unho, my logic was that user with suggest only right can't fix any error.
I wonder if this use case could be:
- translator added wrong translation with critical error;
- suggestor suggested a right translation;
- reviewer accepted the right translation;

Do critical errors need suggestions how they can be fixed?
I didn't think so because there is no possibility to check a suggestion for critical errors. And you will know if there were critical errors in suggestion after you accept it only.